e3db/types/note_options.py

```diff
@@ -0,0 +1,81 @@
+from datetime import date
+
+# Look for a possible alternative to a NoteOptions class. Not a huge priority at the moment, 
+# but this may not be what is wanted. 
+class NoteOptions():
+    
+    def __init__(self, client_id: str, max_views: int, note_name: str, expiration: date, # Check `date` type for expiration. 
+                expires: bool, type: str, plain: dict, file_meta: dict, eacp=None):
+        """
+        Initializes the NoteOptions class. Represents optional values not necessary
+        for creating a note. 
+
+        Parameters
+        ----------
+        client_id : str
+
+        max_views : int
+
+        note_name : str
+
+        expiration : Date
+
+        expires : bool
+
+        eacp : EACP 
+
+        type : str
+
+        plain : dict[string]string
+
+        file_meta : dict[string]string 
+
+        Returns
+        -------
+        None
+        """
+
+        # These may need to be typed properly! 
```

Comment:
Excellent comment, focus on getting the untyped version working first, and then when you have the working solution you can determine both what is worth typing and what the proper types are